python/pyspark/context.py

```diff
@@ -1023,6 +1039,33 @@ def getConf(self):
         conf.setAll(self._conf.getAll())
         return conf
 
+    def install_packages(self, packages, install_driver=True):
+        """
+        install python packages on all executors and driver through pip
+        :param packages: string for single package or a list of string for multiple packages
+        :param install_driver: whether to install packages in client
```

Comment:
Can we have examples as doctests and skip to show how we use it?

Reply:
@HyukjinKwon Could you guide me how to skip such doctests ? Thanks